subsys/mgmt/mcumgr/lib/cmd/os_mgmt/Kconfig

```diff
@@ -121,4 +121,11 @@ endif
 config OS_MGMT_MCUMGR_PARAMS
 	bool "MCUMGR Parameters retrieval command"
 
+config OS_MGMT_UNAME
```

Comment:
Invent your own name for this. Uname is Unix wide command and we have Zephyr here. Make some "os info", "zephyr info" or something. We can use similar option letters as "uname" but command should be ours.

Reply:
Fixed.